data/json/itemgroups/ammo.json

```diff
@@ -53,6 +52,7 @@
     "//": "Imported or otherwise very obscure pistol ammo.",
     "items": [
       [ "454_Casull", 100 ],
+      [ "45colt", 50 ],
```

Comment:
45colt is the amoo_type, not the item, which is 45colt_jhp or reloaded_45_colt_jhp.